smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -334,6 +339,45 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
             await eventHubClient.CloseAsync();
         }
 
+        protected async Task VerifyTwinAsync()
+        {
+            if (this.TwinTestFileName.HasValue)
+            {
+                string twinTestJson = JObject.Parse(File.ReadAllText(this.TwinTestFileName.GetOrElse(string.Empty))).ToString();
+
+                var twinTest = JsonConvert.DeserializeObject<TwinTestConfiguration>(twinTestJson);
+
+                Twin currentTwin = await this.context.RegistryManager.GetTwinAsync(this.context.Device.Id, twinTest.ModuleId);
+
+                if (twinTest.Properties?.Desired != null && twinTest.Properties.Desired.Count > 0)
+                {
+                    string patch = JsonConvert.SerializeObject(twinTest, Formatting.Indented);
```

Comment:
Nit: patch should only include twinTest.Properties?.Desired

Reply:
And I don't think Formatting is needed

Reply:
Took Offline. It was because of the final format of the patch that has to contain properties on the json.